### ISSUE TITLE
trigger sns when some puts object in wasabi bucket

### ISSUE THREAD

**Issue by ahanifztech** [KalobMTaulien/boto3_wasabi#1]
Is there any way to get notfied when someone uploads file to wasabi bucket. like hitting http method, hook or something. thanks

**Comment by KalobTaulien** [comment]
Yep, when the file is done uploading you can check the status of the file by hitting it with a HEAD or GET request. I can't remember the exact method. What I generally do is wrap my upload in a try/except block. 

```python
try:
	upload_file = s3.put_object(
	    Bucket=settings.WASABI_S3_DEFAULT_BUCKET,
	    Key=FILE_KEY,
	    Body=FILE_BODY,
	    ContentType='video/mp4'
	)
except Exception:
	# Upload failed for a reason
	pass
```

If the `upload_file` doesn't throw an error, you can send an email, create a notification in your system, or do anything else you like. 

Here's some documentation from boto3. [https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#uploading-downloading-files-using-sse-kms](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#uploading-downloading-files-using-sse-kms)
